pylabnet/launchers/launcher.py

```diff
@@ -300,16 +300,18 @@ def _connect_matched_servers(self, matches, module, config_name, config, auto_co
                         + '                    ({})                   \n'.format(index + 1)
                         + match.summarize())
                 self.logger.info(msg_str)
-            self.logger.info('------------------------------------------\n\n'
```

Comment:
I think with the new fix, this change should be reverted.

Reply:
Yeah agreed